docs/proposals-accepted/202304-query-path-tenancy.md

```diff
@@ -0,0 +1,80 @@
+---
+type: proposal
+title: Tenancy awareness in query path
+status: accepted
+owner: douglascamata
+menu: proposals-accepted
+---
+
+## Tenancy awareness in query path
+
+* **Owners:**
+  * [@douglascamata](https://github.com/douglascamata)
+
+* **Related Tickets:**
+  * [Querier: Added native multi-tenancy support](https://github.com/thanos-io/thanos/pull/4141).
+  * [Proposal: Native Multi-tenancy Support](https://github.com/thanos-io/thanos/pull/4055)
+
+* **Other docs:**
+  * [Multi-tenancy in Thanos](https://thanos.io/tip/operating/multi-tenancy.md/)
+
+This design doc proposes to add tenancy awareness in the query path.
+
+## Why
+
+In a multi-tenant environment, it's important to be able to identify which tenants are experiencing issues and configure (e.g. with different limits) each one of them individually and according to their usage of the platform so that the quality of service can be guaranteed to all the tenants.
```

Comment:
Nit: the long form is usually preferred for written communication. 

```suggestion
In a multi-tenant environment, it is important to be able to identify which tenants are experiencing issues and configure (e.g. with different limits) each one of them individually and according to their usage of the platform so that the quality of service can be guaranteed to all the tenants.
```